src/Tizen.NUI.Components/Controls/Switch.cs

```diff
@@ -351,16 +351,12 @@ protected override void OnControlStateChanged(ControlStateChangedEventArgs contr
         {
             base.OnControlStateChanged(controlStateChangedInfo);
 
-            if (!IsSelectable)
+            if (IsSelectable)
             {
-                return;
-            }
-
-            bool previousSelected = controlStateChangedInfo.PreviousState.Contains(ControlState.Selected);
-
-            if (previousSelected != IsSelected)
-            {
-                OnSelect();
+                if (controlStateChangedInfo.PreviousState.Contains(ControlState.Selected) != controlStateChangedInfo.CurrentState.Contains(ControlState.Selected))
```

Comment:
As you can see, there's a build warning message here.
`In externally visible method 'void Switch.OnControlStateChanged(ControlStateChangedEventArgs controlStateChangedInfo)', validate parameter 'controlStateChangedInfo' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.`

Probably, controlStateChangedInfo needs to check NULL before use?

Reply:
Thank you for your comment!
I have added the check code.